CRM/Core/Form/Search.php

```diff
@@ -165,6 +172,30 @@ public function addFormFieldsFromMetadata() {
     }
   }
 
+  /**
+   * Format some special form values including date fields
+   *
+   * @param string $name
+   * @param string $value
+   * @param string $dataType
+   *
+   * @return string|array
+   */
+  public function formatSpecialFormValue($name, $value, $dataType) {
+    if ($dataType == 'Date') {
```

Comment:
No, we should convert any date fields to datepicker first - e.g copy what was done on grant search. We don't want to build handling for the old format